doc/_static/mpl.css

```diff
@@ -14,29 +14,6 @@
     margin: 0;
 }
 
-/* Make announcement an error colour for unreleased documentation, and sticky. */
```

Comment:
This CSS interacts badly with dark mode, and removing it (and defaulting to what `pydata-sphinx-theme` uses) looks fine.